### DISCUSSION TITLE
Suggestion to rename contract "category" to something else

### DISCUSSION THREAD

**Discussion by PaulRBerg** [sablier-labs/indexers#14] [Ideas]
How about renaming the `category`:

https://github.com/sablier-labs/v2-subgraphs/blob/main/packages/protocol/schema.graphql#L50-L51

To something else, e.g. `model`?

https://github.com/sablier-labs/v2-core/blob/456744002f62d0627f10b958d41937ed1e68d95e/src/SablierV2NFTDescriptor.sol#L40

Doing so would also create a distinction about this property and [`ActionCategory`](https://github.com/sablier-labs/v2-subgraphs/blob/a387cda7eaa512832b2f1374f88cd3e55fd50297/packages/protocol/schema.graphql#L65C13-L65C27).

**Comment by razgraf** [comment]
We use models throughout the codebases to define classes/entities (extending the `M` concept from MVC) so this renaming would be in conflict. `category` works well in defining the item type, we can continue using it as is.
